applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -204,6 +210,29 @@ static void work_net_core_ctrl_version_get(struct k_work *work)
 	}
 }
 
+static void all_conn_tx_pwr_set(struct bt_conn *conn, void *data)
```

Comment:
This should also be down with the rest of the shell cmd, as this shouldn't be called by any other function, since it will try to print to shell when done or if failing